### PR TITLE
Version number

### PR DESCRIPTION
**Pull request by shuhaib-aot**
1. preview component changed from a class-based to a functional-based component
2. saving version number in the form history table
3. version number is shown in the edit window
4. version details are shown in the form history modal
5. alembic version upgraded

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]

**Review by sreehari-aot — APPROVED** [review]

**Review by sumesh-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/Item/Preview.js [hunk 1]
```diff
@@ -26,43 +26,44 @@ import { INACTIVE } from "../constants/formListConstants";
...
+  const tenantKey = useSelector((state) => state.tenants?.tenantId);
+  const lang = useSelector(state => state.user.lang);
+  const formProcessList = useSelector(state => state.process?.formProcessList);
+
+  const handleModalChange = () => {
+    setHistoryModal(!historyModal);
+  };
 
+  const publishConfirmModalChange = ()=> {
+    setConfirmPublisModal(!confirmPublisModal);
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Form/Item/Preview.js:47]: type Publish

**Review by abhilash-aot — APPROVED** [review]

**Comment by abhilash-aot** [comment]
What about the exiting forms that were created before the introduction of version numbers?

File: forms-flow-web/src/components/Form/Item/Preview.js (hunk 1 above)
Comment (shuhaib-aot) [reply forms-flow-web/src/components/Form/Item/Preview.js:47]: okay

**Comment by shuhaib-aot** [comment]
> What about the existing forms that were created before the introduction of version numbers?
it will be continue like new version created text in form history. and we have new alembic script to add ```new_version:true``` in to initial version of a form so when they do save as new version then it will take count of ```new_version:true``` then will be save with version number in db

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**